general-cookbook-guide.md

```diff
@@ -0,0 +1,107 @@
+# Chef Partner Cookbook Program
+
+## Who should write a Chef Partner Cookbook?
+This document is a specification for the Chef's Partner Cookbook Program. This is
```

Comment:
This kinda doesn't answer the actual question of "who should write a chef partner cookbook?"  Maybe flesh this out a bit:

"As a Chef Partner, we encourage you to publish cookbooks for your solutions and publish your cookbooks via the Chef Partner Cookbook Program. This guide will assist you in that process. This document is a specification..."

"If you have any questions..... Additionally, for more information about becoming a Chef Partner, GO HERE"